deploy/helm/dynatrace-oneagent-operator/templates/Common/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: oneagent
```

Comment:
The secret name appears as configurable on the values file, `{{ .Values.secret.name }}`